charts/prometheus-node-exporter/templates/servicemonitor.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
   {{- end }}
 spec:
   jobLabel: {{ default "app.kubernetes.io/name" .Values.prometheus.monitor.jobLabel }}
+{{ include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | indent 2 }}
```

Comment:
indent looks wrong

Reply:
huh? it's on the same level like `jobLabel`?

Reply:
yes, its part of spec: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.ServiceMonitorSpec